addons/website/views/snippets/s_company_team_detail.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_detail" name="Team Detail">
+    <section class="s_company_team_detail o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <h3>Meet our team</h3>
+            <p class="lead">Dedicated professionals driving our success</p>
```

Comment:
This is usually what we do with heading + lead. It's better structured, easier to resize, you can play with paddings...
```suggestion
        <div class="col-lg-12">
            <h3>Meet our team</h3>
            <p class="lead">Dedicated professionals driving our success</p>
        </div>
```